code/obj/sawflymisc.dm

```diff
@@ -135,18 +139,15 @@
 			else
 				continue
 
-
-
-
-
-
 // ---------------limb---------------
 /datum/limb/sawfly_blades
 
 	//due to not having intent hotkeys and also being AI controlled we only need the one proc
 	harm(mob/living/target, var/mob/living/critter/robotic/sawfly/user) //will this cause issues down the line when someone eventually makes a child of this? hopefully not
-		if(!ON_COOLDOWN(user, "sawfly_attackCD", 1 SECONDS))
-			user.visible_message("<b class='alert'>[user] [pick(list("gouges", "cleaves", "lacerates", "shreds", "cuts", "tears", "saws", "mutilates", "hacks", "slashes",))] [target]!</b>")
+		if(!ON_COOLDOWN(user, "sawfly_attackCD", 0.8 SECONDS))
```

Comment:
?

Reply:
Old code from pre-nerf I missed that shouldn't be there, good catch.